gravitee-apim-console-webui/src/services-ngx/application-user-and-group.service.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Inject, Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import { Constants } from '../entities/Constants';
+import { Member } from '../entities/members/members';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ApplicationUserAndGroupService {
+  constructor(private readonly http: HttpClient, @Inject('Constants') private readonly constants: Constants) {}
+
+  getMembers(applicationId: string): Observable<Member[]> {
+    return this.http.get<Member[]>(`${this.constants.env.baseURL}/applications/${applicationId}/members`);
+  }
+
+  getGroups(): Observable<any[]> {
+    return this.http.get<any>(`${this.constants.env.baseURL}/configuration/groups`);
```

Comment:
I think it's possible to use the existing group service

Reply:
done